resources/sass/original.scss

```diff
@@ -1,27 +1,30 @@
 @import "base";
 
+:root {
```

Comment:
Since these are specific to `original` I deliberately didn't bother putting them in variables. Maybe 1 and 2 should be since they're used more than once...

In that case what do you think of naming them `--gradient-start`/`--gradient-end`?

Reply:
Exactly, I just named them `color1` and `color2` as a temporary placeholder.

I prefer to use `gradient-start` and `gradient-end`, just that since there are two gradients used here, maybe they can be named `primary-gradient-[start|end]` and `secondary-gradient-[start|end]`, I just didn't know which was which when I did it.

Alternatively, would be to make the entire gradient a variable instead. Then, instead of using CSS to apply them to the elements in question, the newly created `--background-image` variable can be used instead.

Reply:
There are 2 gradients but `color3` `color4` and `highlight-color` are only used once so you might as well make them literals

Reply:
The main purpose for using vars instead of literals is being able to override them in the _TBA_ dark theme variant of this theme.